app.rb

```diff
@@ -0,0 +1,10 @@
+letters = [
+  "alpha", "bravo", "charlie", "delta", "echo", 
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.